recipes/tensorflow/samples/tensorflow/retail/retail/tensorflow/evaluator.py

```diff
@@ -0,0 +1,39 @@
+#####################################################################
+# ADOBE CONFIDENTIAL
+# ___________________
+#
+#  Copyright 2019 Adobe
+#  All Rights Reserved.
+#
+# NOTICE:  All information contained herein is, and remains
+# the property of Adobe and its suppliers, if any. The intellectual
+# and technical concepts contained herein are proprietary to Adobe
+# and its suppliers and are protected by all applicable intellectual
+# property laws, including trade secret and copyright laws.
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from Adobe.
+#####################################################################
+from ml.runtime.python.Interfaces.AbstractEvaluator import AbstractEvaluator
+
+import os
+import pickle
+
+class Evaluator(AbstractEvaluator):
+    def __init__(self):
+        print('Initiate')
+
+    def evaluate(self, data=[], model={}, config={}):
+        print('Evaluation evaluate triggered')
+        metrics_dict = pickle.load(open(os.path.join(config['modelPATH'], 'metrics_dict.pkl'), 'rb'))
```

Comment:
might be good to move `metrics_dict.pkl` to a constant

Reply:
`metrics_dict.pkl` is for passing metrics that was computed at the end of trainer to evaluator.

Reply:
I mean to have something like `PICKLE_FILE = 'metrics_dict.pkl'` . But I dont feel too strongly about this.  Either way is ok.

Reply:
Ok, will keep in mind going forward.